### PR TITLE
[13.0][MIG] migrate multi_step_wizard

### PR DESCRIPTION
**Pull request by i-vyshnevska**

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -10,3 +10,4 @@ known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
 default_section=THIRDPARTY
+known_third_party = dateutil,lxml,mock,psycopg2,setuptools
```
Comment (simahawk) [inline .isort.cfg:13]: I'm not sure this is required after `default_section` has been changed.

**Review by simahawk — APPROVED** [review]

**Review by jcoux — APPROVED** [review]
Only technical review: LGTM :+1:

**Review by TDu — APPROVED** [review]

File: .isort.cfg (hunk 1 above)
Comment (i-vyshnevska) [reply .isort.cfg:13]: it was generated by pre-commit, otherwise fails

**Comment by i-vyshnevska** [comment]
@simahawk we can merge here

**Pull request merged**